src/Access/ReplicatedAccessStorage.cpp

```diff
@@ -46,6 +46,7 @@ ReplicatedAccessStorage::ReplicatedAccessStorage(
     , zookeeper_path(zookeeper_path_)
     , get_zookeeper(get_zookeeper_)
```

Comment:
can this be deleted now?

Reply:
No, it's in use, see `ReplicatedAccessStorage::getZooKeeperNoLock()`